sirepo/job_supervisor.py

```diff
@@ -168,16 +168,16 @@ def init_module(**imports):
                 "maximum run-time for sequential job",
             ),
         ),
-        purge_non_premium_after_secs=(
-            0,
-            pkconfig.parse_seconds,
-            "expiration period for purging non-premium users simulations",
-        ),
-        purge_non_premium_task_secs=(
+        purge_check_interval=(
```

Comment:
Please add [ReplacedBy](https://github.com/radiasoft/sirepo/blob/dae139c45ebe9d820c5d8e2e2558d7e0da731353/sirepo/mpi.py#L149) for the old name.

Reply:
796209d1cbcd5b1c7da7d74982d9aeeb0a88e993